### ISSUE TITLE
test

### ISSUE THREAD

**Issue by mowangmodi** [rover-xingyu/Ha-NeRF#11]
1. As for testing, for example, 10 images were used to test the Brandenburg Gate model. Why are these 10 tested images almost free of occlusion? Can I use more occluded images to test?
2. The images obtained from the test are mainly used to remove the occlusion and better learn the light and shadow? Is there any mistake in my understanding?

**Comment by rover-xingyu** [comment]
Yes, the goal is to remove the occlusions but not reconstruct them, so there are no occlusions in the test images.
